debian/rules

```diff
@@ -43,7 +45,6 @@ override_dh_auto_test:
 	cd build && ./tests/unittest_common && cd ..
 	cd build && ./tests/unittest_sink --gst-plugin-path=. && cd ..
 	cd build && ./tests/unittest_plugins --gst-plugin-path=. && cd ..
-	cd build && ./tests/unittest_src_iio --gst-plugin-path=. && cd ..
```

Comment:
Why do you want to remove "unittest_src_iio" in this PR?

Reply:
OMG.. It's a mistake